tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -174,6 +155,18 @@ def dispatch_notification(self, alert):
         server = None
         try:
             self.set_destinations()
+            if (
```

Comment:
ok so I assume the logic below is the answer for https://github.com/Tendrl/notifier/pull/100/files#r133894643

But move this logic there. So, that its easy for anybody to plug a different custom logic for this task and its very modular